README.md

```diff
@@ -1,13 +1,46 @@
 Enable [Yambar] to show [Hyprland] workspaces.
 
+# Installation
+
+Once installed use the Yambar config further down in the README. You'll need
+to adjust the `script` path depending on your installation method.
+
+## Cargo
+
 Assuming you have [Rust installed][Rust], run:
 
 ```console
 $ cargo install yambar-hyprland-wses
 ```
 
-and then use the Yambar config further down in the README. You'll need
-to adjust the `script` path.
+Use `/home/yourusernamehere/.cargo/bin/yambar-hyprland-wses` as your script path
+
+## Arch Linux
+
+There is a community maintained package available in the AUR. Assuming you have
+[yay], run:
+
+```console
+$ yay -S yambar-hyprland-wses
+```
+
+Use `/usr/bin/yambar-hyprland-wses` as your script path.
+
+## Nix
+
+There is a community maintained Nix package on nixpgs. To install add this to
+your Nix configuration
+
+```nix
+environment.systemPackages = [
+  pkgs.yambar-hyprland-wses
+];
+```
+
+Yambar does not check the `$PATH` for scripts which means you cannot just use
+`yambar-hyprland-wses` as the script path. It is probably easiest for Nix users
+to generate the yambar config in your nix config, which will know the correct
+path to `yambar-hyprland-wses`.
```

Comment:
Maybe include a line for how they get that path if they nixify their config?

Reply:
I did a bit of digging and found out how you generate that path in nix config.

I must admit I have barely used Nix so I'm no expert on it. I'm wondering if its worth either:
1. Removing the section on the Nix package as it is mostly a guess, or
2. Simplifying the section to note that there is a nix package if people want it, but you are on your own a little bit with configuration, and PRs for better docs on the Nix package are welcome

Do you have any thoughts?

Reply:
Let's go with option 2 :+1: